ui/src/pageLayout/utils/navItemActivation.ts

```diff
@@ -1,8 +1,14 @@
+// Constants
+import {HOMEPAGE_PATHNAME} from 'src/pageLayout/constants/navItemActivation'
+
 export const getNavItemActivation = (
   keywords: string[],
   location: string
 ): boolean => {
   const ignoreOrgAndOrgID = 3
   const parentPath = location.split('/').slice(ignoreOrgAndOrgID)
+  if (!parentPath.length) {
+    parentPath.push(HOMEPAGE_PATHNAME)
+  }
   return keywords.some(path => parentPath.includes(path))
```

Comment:
i almost feel like includes should also be fixed. cause wouldn't this activate for the url `/new/meme/reviews`?

Reply:
That example has 3 parts and would not activate the homepage icon.

For the other icons, that example would only activate if "reviews" is passed into the 1st argument array.

I have added two additional unit tests to verify the above.

Reply:
above and beyond man!
👍 🚀 🌠